README.md

```diff
@@ -97,11 +97,10 @@ Here is a quick example of how to use Semantic Kernel from a C# console app.
 
 ```csharp
 using Microsoft.SemanticKernel;
-using Microsoft.SemanticKernel.KernelExtensions;
 
 var kernel = Kernel.Builder.Build();
 
-// For Azure Open AI service endpoint and keys please see
+// For Azure Open AI details please see
 // https://learn.microsoft.com/azure/cognitive-services/openai/quickstart?pivots=rest-api
 kernel.Config.AddAzureOpenAICompletionBackend(
```

Comment:
`AddAzureOpenAITextCompletionService`. We can also update service id to `davinci-service`.